aspnetcore/fundamentals/static-files.md

```diff
@@ -89,23 +89,31 @@ Static files are publicly cacheable for 600 seconds:
 
 ## Static file authorization
 
-The Static File Middleware doesn't provide authorization checks. Any files served by it, including those under `wwwroot`, are publicly accessible. To serve files based on authorization:
+The ASP.NET Core templates call <xref:Microsoft.AspNetCore.Builder.StaticFileExtensions.UseStaticFiles%2A> before calling <xref:Microsoft.AspNetCore.Builder.AuthorizationAppBuilderExtensions.UseAuthorization%2A>. Most apps follow this pattern. When the Static File Middleware is called before the authorization middleware:
 
-* Store them outside of `wwwroot` and any directory accessible to the default Static File Middleware.
-* Call `UseStaticFiles` after `UseAuthorization` and specify the path:
+  * No authorization checks are performed on the static files.
+  * Static files served by the Static File Middleware, such as those those under `wwwroot`, are publicly accessible.
+  
+To serve static files based on authorization:
+
+  * Store them outside of `wwwroot`.
+  * Call `UseStaticFiles`, specifying a path, after calling `UseAuthorization`.
+  * Set the [fallback authentication policy](xref:Microsoft.AspNetCore.Authorization.AuthorizationOptions.FallbackPolicy).
 
   [!code-csharp[](static-files/samples/3.x/StaticFileAuth/Startup.cs?name=snippet2)]
   
-  The preceding approach requires users to be authenticated:
-
   [!code-csharp[](static-files/samples/3.x/StaticFileAuth/Startup.cs?name=snippet1&highlight=20-99)]
 
-   [!INCLUDE[](~/includes/requireAuth.md)]
+  The preceding highlighted code sets the [fallback authentication policy](xref:Microsoft.AspNetCore.Authorization.AuthorizationOptions.FallbackPolicy). The fallback authentication policy requires ***all*** users to be authenticated, except for Razor Pages, controllers, or action methods with an authentication attribute. For example, Razor Pages, controllers, or action methods with `[AllowAnonymous]` or `[Authorize(PolicyName="MyPolicy")]` use the applied authentication attribute rather than the fallback authentication policy.
```

Comment:
```suggestion
  In the preceding code, the fallback authorization policy requires ***all*** users to be authenticated. Endpoints such as controllers, Razor Pages, etc that specify their own authorization requirements don't use the fallback authorization policy. For example, Razor Pages, controllers, or action methods with `[AllowAnonymous]` or `[Authorize(PolicyName="MyPolicy")]` use the applied authorization attribute rather than the fallback authorization policy.
```

I dropped the link to the fallback authz policy because it's shown just above this code example. I also changed to use the endpoints terminology with examples so that readers don't think the exceptions list is exhaustive.